lib/main.dart

```diff
@@ -1,18 +1,110 @@
 import 'package:flutter/material.dart';
 
 void main() {
-  runApp(const MyApp());
+  runApp(const CloneInstagramApp());
 }
 
-class MyApp extends StatelessWidget {
-  const MyApp({super.key});
+class CloneInstagramApp extends StatelessWidget {
+  const CloneInstagramApp({super.key});
 
   // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
-    return const MaterialApp(
+    return MaterialApp(
       title: 'Clone Instagram',
+      theme: ThemeData(
+        useMaterial3: true,
+      ),
+      home: const DashboardPage(),
     );
   }
 }
 
+class DashboardPage extends StatefulWidget {
```

Comment:
Should write in a separate file pages/dashboard_page.dart

Reply:
I changed